examples/node/pdf2svg.js

```diff
@@ -20,18 +20,10 @@ var data = new Uint8Array(fs.readFileSync(pdfPath));
 // Dumps svg outputs to a folder called svgdump
 function writeToFile(svgdump, pageNum) {
   var name = getFileNameFromPath(pdfPath);
-  fs.mkdir('./svgdump/', function(err) {
-    if (!err || err.code === 'EEXIST') {
-      fs.writeFile('./svgdump/' + name + "-" + pageNum + '.svg', svgdump,
-        function(err) {
-          if (err) {
-            console.log('Error: ' + err);
-          } else {
-            console.log('Page: ' + pageNum);
-          }
-        });
-    }
-  });
+  if(!fs.existsSync('./svgdump/')) {
+    fs.mkdirSync('./svgdump/')
+  }
+  fs.writeFileSync('./svgdump/' + name + "-" + pageNum + '.svg', svgdump, 'utf-8');
```

Comment:
Change the double quotes to single quotes on this line. Moreover, the prints are now gone. Is there a way to add them back?